packaging/windows/README.md

```diff
@@ -4,10 +4,20 @@ Inspired by [Deluge NSIS installer](https://github.com/deluge-torrent/deluge/blo
 
 ## 1 - Build the application
 
+Build shell extensions that are needed to display icon overlays in windows explorer.
+Make sure you have the windows SDK installed on your local machine (if not install it using the
+Visual Studio installer). You can use `Visual Studio` to build these two dlls __in `Release` mode__
+or invoke `msbuild` from the command line (you have to manually add `msbuild` to your `PATH`).
+
+```shell
+msbuild -maxCpuCount -property:Configuration=Release .\windows-icon-handler\check-icon-handler\check-icon-handler.vcxproj
```

Comment:
Could build the sln instead of each individual vcxproj.

Reply:
I did this in https://github.com/Scille/parsec-cloud/pull/3764/commits/44963686fa2605d2612d273172b6ab691a3b1787 I'll update the `README` guide.